raiden/transfer/node.py

```diff
@@ -110,6 +111,29 @@ def get_token_network_by_token_address(
     return token_network_state
 
 
+def get_token_network_by_address(
+        chain_state: ChainState,
+        token_network_identifier: TokenNetworkID,
+):
+    payment_network_identifier = chain_state.tokenidentifiers_to_paymentidentifiers.get(
```

Comment:
Should this be called `networkaddresses_to_paymentidentifiers`? i think it's more elaborate than `tokenidentifiers` as that would lead me to think it takes `TokenID` or something which doesn't exist.

Reply:
I changed it to `tokennetworkaddresses_to_paymentnetworkaddresses`, the most explicit name I could came up with. This also avoid confusion with `paymentidentifiers` being the value in the message.